core/workspace_svg.js

```diff
@@ -967,6 +989,12 @@ Blockly.WorkspaceSvg.prototype.addZoomControls = function() {
   this.zoomControls_ = new Blockly.ZoomControls(this);
   var svgZoomControls = this.zoomControls_.createDom();
   this.svgGroup_.appendChild(svgZoomControls);
+  this.pluginManager_.addPlugin({
+    id: 'zoomControls',
+    plugin: this.zoomControls_,
+    weight: 0,
+    types: [Blockly.IPositionable]
```

Comment:
Can you check this with compiled mode, and advanced compiled mode? I wonder if interfaces are filtered out completely out of existence that they cause issues here.